src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -91,7 +91,7 @@ def volume_exception_to_retval(self, ve):
         """
         return ve.to_tuple()
 
-    ### volume operations -- create, rm, ls
+    # volume operations -- create, rm, ls
```

Comment:
The reason single # with a space is recommended in pep8 is because someone might think 
################################################################
#                      volume operations -- create, rm, ls                                               #
################################################################
is more emphasizing than 
### volume operations -- create, rm, ls.
Again style guides are just that, guides. If we don't agree, I can close the PR.